generator/src/test/java/com/scottlogic/deg/generator/cucumber/testframework/utils/CucumberGenerationConfigSource.java

```diff
@@ -107,4 +107,9 @@ public boolean visualiseReductions() {
     public GenerationConfig.OutputFormat getOutputFormat() {
         return null;
     }
+
+    @Override
+    public GenerationConfig.OutputDestination getOutputDestination() {
+        return null;
```

Comment:
Favour throwing exceptions for bad usage over passing null. In this case is it possible to split the hierarchy into two subinterfaces, or use an enum to determine if an output destination is necessary?